### DISCUSSION TITLE
boruta integration

### DISCUSSION THREAD

**Discussion by patatoid** [dexidp/dex#3486] [Q&A]
Hi,

Today, I tried to integrate a boruta instance into dex as a connector, it was quite a peaceful path. boruta is an open-source Identity and Access Management solution, the codebase is available on GitHub https://github.com/malach-it/boruta-server.

I recorded a video of the integration, just take the time to watch it, it is short and has great results.

I was wondering if I could publish the video. Also, It would be great for boruta to be listed in the supported OpenID providers, what do you think?

Thank you for reading me so far, hope to hear from you soon.

With care,

## EDIT

Here is a video with the full setup https://www.loom.com/share/bff12e319bbc4ed7b52fbf61b11474c0

**Comment by patatoid** [comment]
up?

**Reply by nabokihms** [reply]
@patatoid sorry for the long delay. May has always been the PTO season. Many maintainers are not available.

I watched the video and it's really great! I think we can repost in on twitter (if you have one).
Boruta can also be mentioned in the doc here https://dexidp.io/docs/connectors/oidc/ as an example of OIDC provider to integrate with.

**Reply by patatoid** [reply]
Hi @nabokihms thank you for your reply.

Do not hesitate to share the demo as you want, I do not have a Twitter account but I would be happy to see it shared there.
It would be great to have a quote about the possible integration of boruta in the docs, I don't know who is able to redact such a section. If you need anything to do so, do not hesitate to reach out.

Thank you once more, would be happy to see boruta integrations with Dex!
